### DISCUSSION TITLE
I want a standardized CAPTCHA

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#131] [Wants]
With the valid and growing privacy concerns about certain common CAPTCHA services, perhaps we should consider an accessible, standardized CAPTCHA. Web developers need a way to help prevent automated bots from spamming their contact forms, while at the same time not frustrating users with uneeded &quot;CAPTCHA hell.&quot;

<hr>

https://webwewant.fyi/wants/43/

**Comment by nikunjbhatt** [comment]
Google's reCaptcha is very reliable to detect robot or human. This should be natively implemented in all browsers. An attribute may be added like `captcha` to make it a captcha like `<input type="checkbox" captcha>` for v2 and `<input type="hidden" captcha>` for v3 kind of reCaptcha.

**Comment by Malvoz** [comment]
See https://github.com/w3ctag/design-reviews/issues/558.

**Comment by PPPDUD** [comment]
It would be extremely hard to enforce such a CAPTCHA, because bots could use their own browsers that ignore CAPTCHAs.

**Reply by nikunjbhatt** [reply]
Captcha needs server side processing as well, so most probably there shouldn't be such an issue. However, as server side processing is required, it would be tricky to configure/implement.
